app/controllers/users.js

```diff
@@ -10,11 +14,46 @@ exports.userCreate = (req, res, next) => {
     password: req.body.password
   };
 
-  User.createUser(user)
+  return User.createUser(user)
     .then(() => {
       res.status(201).end();
     })
     .catch(err => {
       next(err);
     });
 };
+
+const giveToken = req => {
+  const token = jwt.sign(
+    {
+      mail: req.email
+    },
+    secret.common.session.secret
+  );
+  return token;
+};
+
+exports.sesion = async (req, res, next) => {
+  const user = {
+    email: req.body.email,
+    password: req.body.password
+  };
+  try {
+    const result = await User.getUserBy(user.email);
+    if (!result) throw error.signInError('user not registered');
+    return bcrypt.compare(user.password, result.password, (err, validPassword) => {
+      if (validPassword) {
+        const token = giveToken(user);
+        res
+          .cookie('accesToken', token)
```

Comment:
Use a header for return token, `authorization` is normally used (make this a config variable)